src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java

```diff
@@ -16,19 +14,12 @@
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
 public class CategoryTest extends NewTestTemplate {
 
-  private Category category;
-
-  private void init() {
-    this.category = new Category(driver);
-
-    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
-  }
-
-  @Test(groups = "mercury_category_expandAndNavigateToCategoryPage")
-  public void mercury_category_expandAndNavigateToCategoryPage() {
-    init();
-
-    category.toggle();
-    category.openCategoryPage(0);
+  @Test(groups = "mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage")
+  public void mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage() {
+    new MainPage()
+        .openRegularMainPage()
+        .useCategoryComponent()
```

Comment:
@tomasznapieralski suggested that tests should be written from the users perspective, so `get` is not the best choice here. `use` though, indeed doesn't provide much context. I would suggest just breaking the chain here and creating a `new Category` (which could be renamed to `CategoryModule` or `CategoryComponent` BTW) and continue from there.